src/components/Button/Button.tsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import classNames from 'classnames';
 
-import { ButtonProps } from './types';
+import { ButtonProps, ButtonTheme } from './types';
 
 import styles from './Button.css';
 
-export const Button = ({ children, className, ...props }: ButtonProps) => {
+export const Button = ({
+  children,
+  className,
+  theme,
```

Comment:
🧐
Хорошо бы сделать так:
1. Тут используем дефолтное присваивание
```suggestion
  theme = ButtonTheme.Default,
```
2. Делаем мапу тема-класс
```
const ThemeCSSClass = {
  ButtonTheme.Default: styles.buttonThemeDefault,
  ButtonTheme.Glow: styles.buttonThemeGlow,
}
```
3. Присваивание класса темы делаем без условий
```
className={classNames(
        styles.button,
        ThemeCSSClass[theme],
        className
      )}
```

Такой подход спасёт от кучи бойлерплейта в будущем, когда (и если) тем станет больше, и обусловит понятную связь между тем как тема применяется, а именно через css-класс, который хранится в соответствующей мапе

Reply:
Я бы добавил дефолтную тему, но не стал бы вводить дополнительную сущность `ThemeCSSClass`, а просто использовал класс **styles[`buttonTheme${theme}`]**.

Reply:
Добавил дефолтное присваивание, и сами стили сделал как сказал Дима